rules/DowngradePhp80/Rector/ClassMethod/DowngradeAbstractPrivateMethodInTraitRector.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace Rector\DowngradePhp80\Rector\ClassMethod;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\ClassMethod;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\ValueObject\Visibility;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\Tests\DowngradePhp80\Rector\ClassMethod\DowngradeAbstractPrivateMethodInTraitRector\DowngradeAbstractPrivateMethodInTraitRectorTest
+ */
+final class DowngradeAbstractPrivateMethodInTraitRector extends AbstractRector
+{
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition(
+            'Remove "abstract" from private methods in traits and adds an empty function body',
+            [
+                new CodeSample(
+                    <<<'CODE_SAMPLE'
+trait SomeTrait
+{
+    abstract private function someAbstractPrivateFunction();
+}
+CODE_SAMPLE
+                    ,
+                    <<<'CODE_SAMPLE'
+trait SomeTrait
+{
+    private function someAbstractPrivateFunction() {}
+}
+CODE_SAMPLE
+                ),
+            ]
+        );
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [ClassMethod::class];
+    }
+
+    /**
+     * @param ClassMethod $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $node->isAbstract()) {
+            return null;
+        }
+
+        if (! $node->isPrivate()) {
+            return null;
+        }
+
+        // Strip the abstract flag
+        $node->flags -= Visibility::ABSTRACT;
```

Comment:
I think it can be added to `VisibilityManipulator` : 

https://github.com/rectorphp/rector-src/blob/72df0f63c5015db2db86bf1b17a0d8393ad90911/rules/Privatization/NodeManipulator/VisibilityManipulator.php#L107-L110

eg: new method: `removeAbstract()`

Reply:
Agreed!